docs/controller.js

```diff
@@ -50,7 +50,7 @@ const MAZE_STYLE = {
     },
     lines: {
         border: 4,
-        walls: 3,
+        walls: 2,
```

Comment:
Pull upstream changes?

Unless you have a really good reason to make walls thinner...